include/pinocchio/spatial/inertia.hpp

```diff
@@ -889,6 +960,366 @@ namespace pinocchio
+    }
+
+    /**
+     * @brief Converts the PseudoInertiaTpl object to dynamic parameters.
+     * @return A 10-dimensional vector of dynamic parameters.
+     */
+    Vector10 toDynamicParameters() const
+    {
+      Matrix3 I_bar = Sigma.trace() * Matrix3::Identity() - Sigma;
+
+      Vector10 dynamic_params;
+      dynamic_params[0] = mass;
+      dynamic_params.template segment<3>(1) = h;
+      dynamic_params[4] = I_bar(0, 0);
+      dynamic_params[5] = I_bar(0, 1);
+      dynamic_params[6] = I_bar(1, 1);
+      dynamic_params[7] = I_bar(0, 2);
+      dynamic_params[8] = I_bar(1, 2);
+      dynamic_params[9] = I_bar(2, 2);
+
+      return dynamic_params;
+    }
+
+    /**
+     * @brief Constructs a PseudoInertiaTpl object from an InertiaTpl object.
+     * @param inertia An InertiaTpl object.
+     * @return A PseudoInertiaTpl object.
+     */
+    static PseudoInertiaTpl FromInertia(const InertiaTpl<Scalar, Options> & inertia)
+    {
+      Vector10 dynamic_params = inertia.toDynamicParameters();
+      return FromDynamicParameters(dynamic_params);
+    }
+
+    /**
+     * @brief Converts the PseudoInertiaTpl object to an InertiaTpl object.
+     * @return An InertiaTpl object.
+     */
+    InertiaTpl<Scalar, Options> toInertia() const
+    {
+      Vector10 dynamic_params = toDynamicParameters();
+      return InertiaTpl<Scalar, Options>::FromDynamicParameters(dynamic_params);
+    }
+
+    /**
+     * @brief Constructs a PseudoInertiaTpl object from log Cholesky parameters.
+     * @param log_cholesky A 10-dimensional vector of log Cholesky parameters.
+     * @return A PseudoInertiaTpl object.
+     */
+    static PseudoInertiaTpl
+    FromLogCholeskyParameters(const typename LogCholeskyParameters::Vector10 & log_cholesky)
+    {
+      LogCholeskyParameters params(log_cholesky);
+      Vector10 dynamic_params = params.toDynamicParameters();
+      return FromDynamicParameters(dynamic_params);
+    }
+
+    void disp_impl(std::ostream & os) const
+    {
+      os << "  matrix = \n"
+         << matrix << "\n"
+         << "  m = " << mass << "\n"
+         << "  h = " << h.transpose() << "\n"
+         << "  sigma = \n"
+         << Sigma << "";
+    }
+
+    friend std::ostream & operator<<(std::ostream & os, const PseudoInertiaTpl & pi)
+    {
+      pi.disp_impl(os);
+      return os;
+    }
+  };
+
+  /**
+   * @brief A structure representing log Cholesky parameters.
+   *
+   * References:
+   * - Rucker, Caleb, and Patrick M. Wensing. "Smooth parameterization of rigid-body inertia."
+   * IEEE Robotics and Automation Letters 7.2 (2022): 2771-2778.
+   */
+  template<typename Scalar, int Options>
+  struct LogCholeskyParametersTpl
+  {
+    typedef Eigen::Matrix<Scalar, 10, 1, Options> Vector10;
+    typedef Eigen::Matrix<Scalar, 10, 10, Options> Matrix10;
+    typedef PseudoInertiaTpl<Scalar, Options> PseudoInertia;
+
+    Vector10 parameters; ///< 10-dimensional vector of log Cholesky parameters
```

Comment:
Same remark than for PseudoInertiaTpl::matrix.

This is redundant and can lead to some issues.